zcutil/build-debian-package.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+## Usage:
+##  ./build_debian_package.sh
+
+set -e
+set -x
+
+BUILD_PATH="/tmp/zcbuild"
+PACKAGE_NAME="zcash"
+SRC_PATH=`pwd`
+
+umask 022
+
+if [ ! -d $BUILD_PATH ]; then
+    mkdir $BUILD_PATH
+fi
+
+PACKAGE_VERSION=$(grep Version $SRC_PATH/DEBIAN/control | cut -d: -f2 | tr -d ' ')
+BUILD_DIR="$BUILD_PATH/$PACKAGE_NAME-$PACKAGE_VERSION-amd64"
+
+if [ -d $BUILD_DIR ]; then
+    rm -R $BUILD_DIR
+fi
+mkdir -p $BUILD_DIR/DEBIAN $BUILD_DIR/usr/bin $BUILD_DIR/usr/local/bin
+
+cp -r $SRC_PATH/DEBIAN/* $BUILD_DIR/DEBIAN/
+cp $SRC_PATH/src/zcashd $BUILD_DIR/usr/bin/
+cp $SRC_PATH/src/zcash-cli $BUILD_DIR/usr/bin/
+cp $SRC_PATH/zcutil/fetch-params.sh $BUILD_DIR/usr/local/bin/zcash-fetch-params
```

Comment:
Assuming that `build-debian-package.sh` is run as the same user which ran `build.sh` (builds should not be run as root), then this works. Since apt/dpkg is run as root, they are installed as root-owned binaries, chmod 755.